hal_nxp has invalid key "click-through" in module.yml

### Describe the bug

hal_nxp/zephyr/module.yml contains an invalid key "click-through" causing the below issue when running `west build`. Not entirely sure when this was introduced but saw this on version included in west.yml from 4.2.0 and 4.3.0-rc1.

```
CMake Error at zephyr/zephyr/cmake/modules/zephyr_module.cmake:73 (message):
  validation.invalid

   --- All found errors ---

  ["Key 'click-through' was not defined.  Path: '/blobs/22'"]

  ERROR: Malformed "build" section in file:
/<some dir>/project/zephyr/modules/hal/nxp/zephyr/module.yml

  <SchemaError: error code 2: Schema validation failed:

   - Key 'click-through' was not defined. Path: '/blobs/22'.: Path: '/'>

Call Stack (most recent call first):
  zephyr/zephyr/cmake/modules/zephyr_default.cmake:133 (include)
  zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:92 (include_boilerplate)
  CMakeLists.txt:5 (find_package)
```

### Regression

- [x] This is a regression.

### Steps to reproduce

build any project for any nxp board.

### Relevant log output

```shell

```

### Impact

Annoyance – Minor irritation; no significant impact on usability or functionality.

### Environment

- OS: Linux but would happen on any OS.
- zephyr-sdk-0.16.3
- 

### Additional Context

_No response_

--- a/scripts/west_commands/blobs.py
+++ b/scripts/west_commands/blobs.py
@@ -186,7 +186,7 @@ class Blobs(WestCommand):
                     self.wrn('Skip fetching this blob.')
                     continue
 
-                with open(blob['license-abspath']) as license_file:
+                with open(blob['license-abspath'], encoding="utf-8") as license_file:
                     license_content = license_file.read()
                     print(license_content)
 

